formspree/forms/models.py

```diff
@@ -312,3 +312,10 @@ def __init__(self, form_id):
     def __repr__(self):
         return '<Submission %s, form=%s, date=%s, keys=%s>' % \
             (self.id or 'with an id to be assigned', self.form_id, self.submitted_at.isoformat(), self.data.keys())
+
+    @classmethod
+    def get_by_submissionid(cls, submissionid):
+        try:
+            return Submission.query.filter_by(id=submissionid).first()
+        except IndexError:
+            return None
```

Comment:
is an `IndexError` ever raised here? `.first()` normally returns `None` when no records are found in the query. It is not the same as calling `.all()[0]`.